Marlin/src/gcode/feature/powerloss/M1000.cpp

```diff
@@ -64,6 +68,15 @@ void GcodeSuite::M1000() {
         SERIAL_ECHO_MSG("Resume requires LCD.");
       #endif
     }
+    else if (parser.seen('C')) {
+      #if HAS_LCD_MENU
+        lcd_power_loss_recovery_cancel();
+      #elif ENABLED(EXTENSIBLE_UI)
+        card.removeJobRecoveryFile();
+        card.autostart_index = 0;
+        ExtUI::onPrintTimerStopped();
+      #endif
```

Comment:
Should this be set up to do both, for the case where there are both regular LCD and ExtUI devices attached?

Reply:
Ok, I set it up to handle the case where both are attached.

Reply:
Good catch following the discussion the other day. I obviously wasn't thinking about it earlier today!